doc/fileformat/nmodl.rst

```diff
@@ -159,6 +159,48 @@ Arbor-specific features
   made available through the ``v_peer`` variable while the local membrane potential
   is available through ``v``, as usual.
 
+
+.. _format-sde:
+
+Stochastic Processes
+--------------------
+
+Arbor supports :ref:`stochastic processes <mechanisms-sde>` in the form of stochastic differential
+equations. The *white noise* sources can be defined in the model files using a `WHITE_NOISE` block:
+
+.. code:: none
+
+   WHITE_NOISE {
+       SEED = 42
+       a b 
+       c
+   }
+
+The ``SEED`` keyword defines an optional seed value which must be a positive integer.
+This value is useful for testing if one would like to prevent varying results from run to run.
+The seed  value can also be set using the mechansim parameter ``seed``, e.g. ``"my_mech/seed=42"``.
+If the seed value remains unset, the random number generator will be initialized from a
+stochastic system resource at runtime.
```

Comment:
This approach requires that in order to get reproducible results from run to run, the user has to explicitly specify a seed for every single mechanism that is added to a cell.

The preferable default behavior (in my opinion), is that the default behavior is always reproducible, which would 
* the seed on every mechanism is always, by default, set to a fixed global value (e.g. 1)
* the global seed can be changed explicitly by the user, which would affect all mechanism instances.
* if a user wants to use a stochastic system resource, they can use it to set the global seed, or maybe override the seed at the mechanism level.

With this in mind, one would not be able to specify a seed in the NMODL mechanism description (i.e. remove the `SEED` keyword).
We could also think about not having seed be specialized for each mechanism instance (i.e. all mechanisms always use the same global seed).

Reply:
I removed the `SEED` keyword. The seed can now be set as part of the recipe.

Reply:
Changed the see to be part of the simulation setup (constructor).